extensions/git/src/commands.ts

```diff
@@ -671,6 +677,29 @@ export class CommandCenter {
 		await this.model.branch(name);
 	}
 
+	@command('git.merge')
+	async merge(): Promise<void> {
+		const config = workspace.getConfiguration('git');
+		const checkoutType = config.get<string>('checkoutType') || 'all';
+		const includeRemotes = checkoutType === 'all' || checkoutType === 'remote';
+
+		const heads = this.model.refs.filter(ref => ref.type === RefType.Head)
+			.map(ref => new CheckoutItem(ref));
+
+		const remoteHeads = (includeRemotes ? this.model.refs.filter(ref => ref.type === RefType.RemoteHead) : [])
+			.map(ref => new CheckoutRemoteHeadItem(ref));
+
+		const picks = [...heads, ...remoteHeads];
+		const placeHolder = 'Select a ref to checkout';
```

Comment:
Where's the `localize` call? Remember, Code needs to be localized for other languages. Also, the sentence is wrong. It should be something like `Select a branch to merge from`